slm_lab/agent/memory/replay.py

```diff
@@ -113,7 +113,7 @@ def sample(self):
     def sample_idxs(self, batch_size):
         '''Batch indices a sampled random uniformly'''
         batch_idxs = np.random.choice(list(range(self.true_size)), batch_size - 1)
-        batch_idxs = np.append(batch_idxs, self.true_size - 1)  # add the latest sample
+        batch_idxs = np.append(batch_idxs, self.true_size - 1)  # add the latest sample for CER
```

Comment:
I know it would have been helpful for me to have weighed in on this earlier, but I don't think we should be doing CER by default because it is non-standard. At least we should have the option to toggle it on and off. 

What do you think about this?
batch_idxs = np.random.choice(list(range(self.true_size)), batch_size)
if CER:
    batch_idxs[-1] = self.true_size - 1  # add the latest sample for CER